### PR TITLE
Portfolio: mobile version finished 

### PR DESCRIPTION
**Pull request by Alexr16**
Works section added.
About myself added.
Elements in Works section lay outed with CSS Grid and Flexbox combined.
`ul` and `li` used for listed categories in About myself section.

### CODE REVIEW

**Review by MrBrN197 — COMMENTED** [review]
## Hi, @Alexr16  👋 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that that has linter errors. please fix the linter errors and then ask for a re-review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Alexr16** [comment]
linters errors solved.

File: styles.css [hunk 1]
```diff
@@ -100,18 +104,297 @@ h1 {
...
+  font-family: Inter, sans-serif;
+  font-size: 17px;
+  line-height: 24px;
+  text-align: center;
+  border: none;
+}
+
+.button:hover {
+  background-color: #e97f42;
+}
```
Comment (elmar8287) [inline styles.css]: - [ ] Please, consider that the hover should be as follows, as in the Figma template:
```
.button:hover {
  background-color: #FFF5E1;
  color: #F55800;
  border: 1px solid #F55800;
  box-sizing: border-box;
}
```

File: index.html [hunk 2]
```diff
@@ -56,7 +52,161 @@ <h4>I’am a Software Developer</h4>
...
+                        sign-ups required. has been the
+                        industry's standard dummy text ever since the 1500s, when an unknown printer took a standard
+                        dummy
+                        text.</p>
+                    <div class="tag-container">
+                        <span class="tag">css</span>
+                        <span class="tag">html</span>
+                        <span class="tag">bootstrap</span>
+                        <span class="tag">Ruby</span>
+                    </div>
```
Comment (elmar8287) [inline index.html]: - [ ] Please, consider that the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `ul` and each of them a `li`

File: index.html [hunk 3]
```diff
@@ -56,7 +52,161 @@ <h4>I’am a Software Developer</h4>
...
+                    <div class="card-button">
+                        <button class="button button-display">See Project</button>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section class="about-me">
+        <div class="about-info">
+            <h1>About me</h1>
```
Comment (elmar8287) [inline index.html]: - [ ] Please, consider that you could have one `h1` on your webpage. All other headers should have `hx` tag (aka h1, h2, h3, etc..) - try to replace it everywhere in your code.

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi @Alexr16, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  Check the comments under the review.
- [ ]  Please, consider fixing the horizontal scrolling on your page. It seems, the issue is with the background.



## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Alexr16** [comment]
Horizontal scrolling fixed repaired.
Hover changed.
List of project technologies (e.g., "JavasScript", "HTML" and "CSS") made with list `ul` and `li`.
Header tags `hx` changed.

**Review by codepantha — CHANGES_REQUESTED** [review]
Hi @Alexr16 👋 , 

Great work on making most of the changes requested by the previous reviewer 👏 
but there is still one that has not been fully addressed.

## Suggested changes

- [ ] Though I appreciate your effort in trying to fix the horizontal scrolling issue, it hasn't been fully resolved yet. Kindly look at this [resource](https://www.codegrepper.com/code-examples/css/html+fix+horizontal+scroll) for a pointer to what you may be missing

image

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [ ] It would be nice to have detailed steps to clone a local copy of your repo.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by Alexr16** [comment]
Horizontal issue scrolling issue solved.

**Review by Lordkaito — APPROVED** [review]
Hi @Alexr16, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

- [ ] [OPTIONAL] Even though you mostly solved the horizontal scrolling, there is a still some when the screen is not big enough, so I would recommend to apply this properties to your `headline-ilustration` container:
```
headline-ilustration {
  display: flex;
  align-items: center (or you can use flex-end as well, not a big deal here)
  justify-content: flex-end (here it is necessary to apply flex-end)
}
```
And also apply this to the image inside this container:
```
.headline-img {
  width: 60%;
  /* margin: auto; */ (you can remove this one)
  position: relative;
  /* left: 190px; */ (you need to remove this one)
  top: 160px;
}
``` 
With that your problem should be completely gone!

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**